rfcs/text/0008-threat-intel.md

```diff
@@ -174,6 +448,7 @@ The following are the people that consulted on the contents of this RFC.
 * Stage 0: https://github.com/elastic/ecs/pull/986
 * Stage 1: https://github.com/elastic/ecs/pull/1037
   * Stage 1 correction: https://github.com/elastic/ecs/pull/1100
+* Stage 2: https://github.com/elastic/ecs/pull/1127
```

Comment:
```suggestion
* Updated Stage 1 (formerly Stage 2): https://github.com/elastic/ecs/pull/1127
```

Also, clarify here that this PR was formerly targeting stage 2 until the RFC stages were altered.